index.html

```diff
@@ -57,11 +57,227 @@ <h3 class="connect">LET'S CONNECT</h3>
+                <li>JAVASCRIPT</li>
+              </ul>
+
+              <a class="btn" href="#">See projects</a>
+            </div>
+          </div>
+
+          <div class="details">
+            <img src="./images/Snapshoot-Portfolio1.png" alt="" />
+
+            <div class="card-info">
+              <h2>Multi-Post Stories</h2>
+
+              <div class="title-and-date">
+                <p class="title">CANOPY</p>
+                <span></span>
+                <p class="muted">BACK END DEV</p>
+                <span></span>
+                <p class="muted">2015</p>
+              </div>
+
+              <p class="description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+
+              <ul class="stack">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+
+              <a class="btn" href="#">See projects</a>
+            </div>
+          </div>
+
+          <div class="details">
+            <img src="./images/Snapshoot-Portfolio2.png" alt="" />
+
+            <div class="card-info">
+              <h2>TONIC</h2>
+
+              <div class="title-and-date">
+                <p class="title">CANOPY</p>
+                <span></span>
+                <p class="muted">BACK END DEV</p>
+                <span></span>
+                <p class="muted">2015</p>
+              </div>
+
+              <p class="description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+
+              <ul class="stack">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+
+              <a class="btn" href="#">See projects</a>
+            </div>
+          </div>
+
+          <div class="details">
+            <img src="./images/Snapshoot-Portfolio3.png" alt="" />
+
+            <div class="card-info">
+              <h2>Multi-Post Stories</h2>
+
+              <div class="title-and-date">
+                <p class="title">CANOPY</p>
+                <span></span>
+                <p class="muted">BACK END DEV</p>
+                <span></span>
+                <p class="muted">2015</p>
+              </div>
+
+              <p class="description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+
+              <ul class="stack">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+
+              <a class="btn" href="#">See projects</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- MYSELF -->
+    <section class="about-bg">
+      <div class="myself about container">
+        <h1>About Myself</h1>
```

Comment:
**OPTIONAL** - To help improve the SEO of your site, it is not advisable to have more than one `h1` tag in an HTML page, please fix this 🙏. And also have it in mind to follow the proper `hx` hierarchy i.e an `h2` tag should be used after an `h1` tag e.tc.

Reply:
Well noted. Thanks